### DISCUSSION TITLE
Git signs column

### DISCUSSION THREAD

**Discussion by marendowski** [gokcehan/lf#1781] [Q&A] [answered]
Hi, I would like to set up a column with git signs next to the icons, like I did in [my fff fork](https://github.com/bogdan-the-great/fff):

![image](https://github.com/user-attachments/assets/8188cbaf-8cd8-41f3-a0a6-dad773dcc0dc)
(plus signs next to changed files)

Is this possible with `lfrc` only?

**Comment by marendowski** [comment]
Closing as duplicate, because #83 and other exist.

**Reply by CatsDeservePets** [reply, marked as answer]
For anyone in the future reading this, this is now possible thanks to 
- https://github.com/gokcehan/lf/pull/2012

You can find an example configuration [here](https://github.com/gokcehan/lf/blob/master/doc.md#loading-directory)!
